runner/docker/Dockerfile.mediamtx

```diff
@@ -0,0 +1,7 @@
+FROM bluenviron/mediamtx:1.9.0-ffmpeg
```

Comment:
So, we plan to run media server as a separate Docker container and then send video over the HTTP into ai-runner container. Is that correct?

Asking to understand it better. I think it's fine. It'll both run locally, so AFAIU the only additional time would be to segment the video and send it over HTTP POST Form Data.

Reply:
But I guess this PR is for MVP-phase and it's already running mediamtx as a separate Docker container. So, do I understand correctly that even for MVP we plan to run mediamtx and ai-runner in two separate containers?

Reply:
We had some offline discussions around this and summary is one ai-runner like docker container that will do the following: 
```
mediamtx (ingress) -> decode video -> encode image -> inference -> encode video -> mediamtx (egress)
```
Post MVP, we might move to HTTP streaming workflow which would require swapping out the mediamtx bits but for now, it serves it's purpose to allow ingress/egress capabilities to the ai-runner container. Having separate containers creates more moving parts along with more integration code across boundaries - let's keep it simple for now and evolve it as we build. 

So we'll jam in everything in this new container that Josh started.

Reply:
Follow-up comment: we might have to move to a two container approach post-MVP. One thing we're seeing is the requirements/dependencies of the model and the requirements/dependencies of the rest of the pipeline (video-decode -> image-encode etc) might conflict with one another. 

On the flip side, there's also the idea that if we can keep everything within GPU memory (zero-copy), then that will be better suited to a single container. But then we violate the docker "one concern per container" which is not a hard and fast rule and is generally a recommendation -- we can easily use a supervisor daemon process to manage lifetime of the individual components of the realtime pipeline. 

For now, it's easiest to keep it in one container and continue building for the MVP. Once we have it in place, we should have more info on which path to take.

Reply:
It all makes sense. The thing I'm confused about is that this PR implements the two-container solution. While you mentioned about that the MVP solution is one-container 🤔